bin/deploy-to-wp-org.sh

```diff
@@ -8,8 +8,14 @@ function deploy () {
 	git clone git@github.com:Upstatement/timber-starter-theme.git
 	rm -rf ~/Sites/timber/timber-starter-theme/.git
 	rm composer.lock
-	composer config platform.php 5.6.20
+
+	# Download dependencies for the maximum compatible PHP version.
+	composer config platform.php 8.0
```

Comment:
We should actually do the opposite. Install dependencies for the minimum compatible version.

`composer config platform.php 7.2.5`

Otherwise some dependency in the dependency tree might not be compatible with PHP 7.2.5. Although the opposite is also true for PHP 8.0 (some dependency might not be compatible with PHP 8), however, the consequences should not be as dramatic, e.g. a fatal error (VS a deprecation warning).

Reply:
Oh yeah, of course, that makes sense. I updated it in [`f3d92d6` (#2658)](https://github.com/timber/timber/pull/2658/commits/f3d92d6e72893c9fd8703efbef3bb49acc6d5183).

I think we better stay conservative here. People who really want to upgrade to the latest version of PHP can do so using Composer.